lib/google/ads/google_ads/resource.rb

```diff
@@ -0,0 +1,1566 @@
+module Google
+  module Ads
+    module GoogleAds
+      def self.resource
+        @resource ||= Module.new do
+        
+          def self.http
+            require "google/api/http_pb.rb"
+            Google::Api::Http.new
+          end
+        
+          def self.http_rule
+            require "google/api/http_pb.rb"
+            Google::Api::HttpRule.new
+          end
+        
+          def self.custom_http_pattern
+            require "google/api/http_pb.rb"
+            Google::Api::CustomHttpPattern.new
+          end
+        
+          def self.double_value
+            require "google/protobuf/wrappers_pb.rb"
```

Comment:
Having access to these wrapper things will be moot once the new version of the protobuf library is out since the objects will support `*_as_value` methods, so something like this would work: `campaign.name_as_value = "new name"`.

We can still expose them in case people want to explicitly use wrappers but I'd also move these to their own module with the google/api stuff.